extension/src/background/helpers/session.ts

```diff
@@ -14,11 +11,10 @@ export class SessionTimer {
     this.DURATION = duration || this.DURATION;
   }
 
-  startSession(key: { privateKey: string }) {
+  startSession() {
     if (this.runningTimeout) {
       clearTimeout(this.runningTimeout);
     }
-    store.dispatch(grantAccountAccess(key));
```

Comment:
I decoupled starting a session w/ setting the private key. Since now we'll start a session if confirming password while using a hardware wallet, but there's no private key

for this change I also changed the name from `grantAccountAccess` to `setPrivateKey`